### ISSUE TITLE
inline_templates only evaluate in double quotes, yet lint warns of no vars.

### ISSUE THREAD

**Issue by adamgibbins** [rodjek/puppet-lint#94]
``` ruby
$foo = 'one|two|three'
$test = inline_template("<%= foo.split('|')[0] %>")
notify { $test: }
```

Produces:

> notice: one
> notice: /Stage[main]//Notify[one]/message: defined 'message' as 'one'
> notice: Finished catalog run in 0.02 seconds

Puppet lint complains:

> WARNING: double quoted string containing no variables on line 2

To fix lint warning, I place line 2 in single quotes which results in:

> Could not parse for environment production: Could not match |')[0] at foo.pp:2 on node example.com

As expected.

inline_templates need to be in double quotes for them to be parsed.  Lint should not warn in this circumstance.
